identity-apps-core/apps/accounts/src/main/resources/org/wso2/carbon/identity/application/accounts/endpoint/i18n/Resources.properties

```diff
@@ -0,0 +1,424 @@
+authentication.flow.timeout=Sorry! Your sign-in attempt was taking too long
+authentication.flow.timeout.description=Looks like the sign-in process ended because of the delay. Let's start again. If this problem persists, please contact the administrator.
+authentication.flow.app.disabled=Sorry! Application is disabled.
+authentication.flow.app.disabled.description=Looks like the application is disabled and the access to the application is restricted. Please contact your administrator.
+tracking.reference.id=Tracking Reference ID
+copied=Copied
+select.language=Select Language
+you.are.all.set=You're all set!
+click=Click
+account=account
+to.sign.in.with=to sign in with
+need.help.contact.us=Need help? Contact us via
+with.tracking.reference.below=with the tracking reference given below.
+could.not.retrieve.email=Couldn't Retrieve an Email
+seems.like.private.email.in.github=Seems you have kept email addresses private in the GitHub profile or haven't configure a public email.
+plase.configure.public.email.github=Please configure a public email to continue Sign in with GitHub. Check
+email.settings=email settings
+of.your.profile=of your profile.
+this.will.allow.application.to=This will allow application to
+read.your.profile=Read your profile
+by.giving.consent.you.agree.to.share.data=By giving consent, you agree to share your data with
+app=App
+dont.have.app.download.google.authenticator=Don't have an app? Download an authenticator application like Google Authenticator from
+if.you.still.havent.received.please.write.to=If you still haven't received it, please write to
+for.assistance=for assistance.
+please.enter.mobile.number=Please enter the mobile number!
+something.went.wrong=Something Went Wrong
+something.went.wrong.contact.administrator=Something went wrong during the authentication process. Please contact administrator.
+logout=Logout
+are.you.sure.that.you.want.to.logout=Are you sure that you want to logout?
+click.here=Click here
+if.you.have.been.waiting.for.too.long=if you have been waiting for too long.
+complete.your.profile=Complete Your Profile
+needs.following.details.that.are.missing.in.profile=needs the following details that are missing in your profile.
+dob.yyyy.mm.dd=Date of Birth (YYYY-MM-DD)
+your.account.is.locked.write.us.to.know.more.at=Your account is locked at the moment. If you want to know more, please write to us at
+your.account.is.disabled.write.us.to.know.more.at=Your account is disabled at the moment. If you want to know more, please write to us at
+redirecting=Redirecting
+please.wait.while.we.take.you.back.to=Please wait while we take you back to
+account.not.found=Account Not Found
+we.did.not.receive.your.email.address=We did not receive your email address.
+please.use.a.social.account.with.email.available.to.sign.in=Please use a social account with the email available to sign in.
```

Comment:
Let's review and remove all unnecessary i18n content from the new app, as most of it is only required for the authentication endpoint.